ml-agents/tests/yamato/scripts/run_gym.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+from gym_unity.envs import UnityEnv
+
+
+def main():
+
+    # Name of the Unity environment binary to launch
+    multi_env_name = "Project/testPlayer"
```

Comment:
Should this be a global variable to be used by both run_gym.py and run_llapi.py ?

Reply:
I agree, either global variable or passing it to where it is need would be better than keeping it around multiple places.

Reply:
It should probably be a command line argument that defaults to this.